documentation/release-notes/topics/version_3.rst

```diff
@@ -3,6 +3,107 @@
 #########
 Version 3
 #########
+**********************************
+Hyperview 3.11 (February 15, 2023)
+**********************************
+This section covers significant changes and bug fixes in Hyperview 3.11.x since version 3.10.x.
+
+.. note:: This release has changes and improvements to the Data Collector software. It is strongly recommended to update your installed Data Collectors to the latest version to maintain an optimal monitoring and discovery experience.
+
+=============================================
+New Licensed Feature: Rack PDU Outlet Control
+=============================================
+Outlet Control is a new licensed feature that allows users to:
+
+- Power on
+- Power off
+- Power cycle
+
+Supported Rack PDU hardware.
+
+.. note:: Please contact our Sales team if you are interested in getting an Outlet Control license.
+
+.. warning:: The latest Data Collector software is needed to take advantage of this feature.
+
+========================================
+Enhanced Licensed Feature: Rack Security
+========================================
+Rack Security control actions have been enhanced to integrate with Work Orders. A rack open operation will create a corresponding Work Order to document and track the action.
+
+.. note:: Please contact our Sales team if you are interested in getting a Rack Security license.
+
+.. warning:: The latest Data Collector software is needed to take advantage of this feature.
+
+===============================================
+Enhanced Licensed Feature: ServiceNow CMDB Sync
+===============================================
+Administrators are now able to test the ServiceNow credentials from the configuration page before any sync job is attempted.
+
+.. note:: Please contact our Sales team if you are interested in getting a ServiceNow CMDB Integration license.
+
+============================================
+New Feature: Automated Catalog Data Feedback
+============================================
+A new feature was added to enable automated feedback to Hyperview when previously unknown asset models are added. This feature will allow Hyperview to add support to new models more rapidly.
```

Comment:
I'd say "A new feature was added to enable automatic submission of model data to Hyperview when a previously unknown asset model is created"